src/Plugin/RulesAction/SystemMessage.php

```diff
@@ -37,14 +37,20 @@
 class SystemMessage extends RulesActionBase {
 
   /**
-   * {@inheritdoc}
+   * Set a system message.
+   *
+   * @param string $message
+   *   Message string that should be set.
+   * @param string $type
+   *   Type of the message.
+   * @param bool $repeat
+   *   Boolean if the message should be repeated.
```

Comment:
should be prefixed with "(Optional)". Comment should be "TRUE if the message should be repeated."